pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -651,7 +731,8 @@ func (p *awsSDKProvider) Compute(regionName string) (EC2, error) {
 		Region:      &regionName,
 		Credentials: p.creds,
 	}
-	awsConfig = awsConfig.WithCredentialsChainVerboseErrors(true)
+	awsConfig = awsConfig.WithCredentialsChainVerboseErrors(true).
+		WithEndpointResolver(endpoints.ResolverFunc(p.cfg.getResolver()))
```

Comment:
You don't need the `endpoints.ResolverFunc()` wrapper, can you just do 

```go
awsConfig =  awsConfig.WithCredentialsChainVerboseErrors(true).
        WithEndpointResolver(p.cfg.getResolver())
```

Reply:
yes, not needed since I changed the getResolver signature. That cast came from the AWS example.